src/app/screens/Accounts/AdvancedSettings/index.tsx

```diff
@@ -12,35 +12,35 @@ import Button from "~/app/components/Button";
 import InputCopyButton from "~/app/components/InputCopyButton";
 import TextField from "~/app/components/form/TextField";
 import { useAccount } from "~/app/context/AccountContext";
+import { KeyOrigin, getKeyOrigin } from "~/app/utils/getKeyOrigin";
```

Comment:
maybe this file/component should be called `AdvancedKeySettings` or something. It's for Nostr and Liquid key management, right?

Reply:
Yup was thinking the same, will change 👍